index.js

```diff
@@ -1,53 +1,42 @@
 const dogs = []
-document.addEventListener('DOMContentLoaded', () => {
-    fetch('http://localhost:3000/dogs') 
+document.addEventListener("DOMContentLoaded", () => {
+    fetch("http://localhost:3000/dogs")
     .then(response => response.json())
     .then(data => getDogs(data))
    
-    const searchForm = document.getElementById("search-form");
+    const searchForm = document.getElementById("search-form")
     searchForm.addEventListener("submit", (e) => {searchDogs(e)})    
    
     const allDogsButton = document.getElementById("all")
-    allDogsButton.addEventListener("click", () => {showAllDogs()})
-    
+    allDogsButton.addEventListener("click", () => {
+        const dc = document.getElementById("dogs-container")
```

Comment:
I noticed that if I clicked the All Dogs button, it added the dogs. Then if I clicked it again...it added the dogs again. So to prevent having duplicates, I wrote this if statement.

In plain english, if the dogs-container has no children (elements inside of it), add the dogs.

Reply:
I don't see the if statement...

Reply:
Oh, NVM it's bellow